libs/@local/effect-dns/hickory/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@local/effect-dns-hickory",
+  "version": "0.0.0-private",
+  "private": true,
+  "description": "Hickory DNS implementation of DNS effect library",
+  "license": "MIT OR Apache-2.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "rimraf dist && yarn build:release && tsc --build tsconfig.build.json",
+    "build:debug": "napi build dist --platform --js=impl.js --dts=impl.d.ts",
+    "build:release": "napi build dist --platform --release --js=impl.js --dts=impl.d.ts",
+    "fix:eslint": "eslint --fix .",
+    "lint:eslint": "eslint --report-unused-disable-directives .",
+    "lint:tsc": "tsc --noEmit",
+    "regenerate-packages": "napi create-npm-dir --target=.",
```

Comment:
I don't find a documentation for this command online. What does it? When it is invoked?

Reply:
This is in preparation for if we want to publish. Not needed when in-tree, but when publishing we need to create a package for _every single target_, which is then installed depending on the architecture. `create-npm-dir` creates the packages and directory structure required for publishing.

Reply:
Makes sense, thanks for explaining!